### PR TITLE
feat: Support apple cert root file path in param

### PR DESCRIPTION
**Pull request by jawad-khan**
Adding apple cert root file path in param gives more flexibity in program. 
Bumped versions of cryptography to 40.0.2 and PyJWT to 2.7.0

### CODE REVIEW

File: app_store_notifications_v2_validator/__init__.py [hunk 1]
```diff
@@ -16,18 +16,24 @@ def add_labels(key: str) -> bytes:
...
 
-fn = os.environ.get("APPLE_ROOT_CA", "AppleRootCA-G3.cer")
-fn = os.path.expanduser(fn)
-with open(fn, "rb") as f:
-  data = f.read()
-  root_cert = load_certificate(FILETYPE_ASN1, data)
+def _get_root_cert(root_cert_path):
 
+  fn = os.environ.get("APPLE_ROOT_CA")
+  if not fn:
```
Comment (rickwierenga) [inline app_store_notifications_v2_validator/__init__.py]: `if fn is not None`

**Review by rickwierenga — CHANGES_REQUESTED** [review]

**Comment by rickwierenga** [comment]
Thanks! See one minor thing I think could be improved.

File: app_store_notifications_v2_validator/__init__.py (hunk 1 above)
Comment (jawad-khan) [reply app_store_notifications_v2_validator/__init__.py]: updated.

**Comment by rickwierenga** [comment]
Thanks again! I'll update the PyPI version.

**Comment by jawad-khan** [comment]
Thanks, it was quick.

**Comment by rickwierenga** [comment]
All you :)

**Pull request merged**